### ISSUE TITLE
add development fixtures

### ISSUE THREAD

**Issue by brylie** [CiviWiki/OpenCiviWiki#1350]
Model factories can help us scaffold content quickly for development purposes.

Let's define a `factory.py` file in every app containing a `models.py`. The `factory.py` files will contain factories for the models in the same app. E.g.,

- project/categories/models.py - model definitions for categories app
- project/categories/factory.py - factory definitions for categories app

## Task

- [ ] add `factory_boy` to this project with `poetry add factory_boy --group dev`
- [ ] define the [initial factory_boy factories](https://factoryboy.readthedocs.io/en/stable/index.html?highlight=install#defining-factories)

**Comment by Walexero** [comment]
Would love to work on this issue, and would need your help with completing the second requirements

**Comment by brylie** [comment]
Ok. Let's start with one issue at a time for now.

**Comment by Walexero** [comment]
okay, would start working in it, would, would make  a pull request now

**Comment by Walexero** [comment]
Have already added factory boy into the project with the provided command, can you expatiate on the second task

**Comment by Walexero** [comment]
do you mean to define or decline?

**Comment by brylie** [comment]
I meant "define". Good catch.

**Comment by Walexero** [comment]
is there a specific folder you want me to define the initial factory_boy factory into and a specific filename to be used. Please expatiate

**Comment by brylie** [comment]
Let's define a `factory.py` file in every app where we also have a `models.py`. The `factory.py` files will contain factories for the models in the **same** app. E.g.,

- `project/categories/models.py` - model definitions for `categories` app
- `project/categories/factory.py` - factory definitions for `categories` app

**Comment by Walexero** [comment]
okay, would get it done now

**Comment by Nehemiah60** [comment]
Hello, @brylie. I will wait in line if this issue is still open. I would wish to be assigned this. Thanks.

**Comment by Nehemiah60** [comment]
Hello, @brylie. I created the factories model factories for one app first. I want to merge changes, get your feedback, and then I can go on step by step. I think that will be easier until I manage all the apps. Thanks. 
And also I made a PR 3 days ago on template porting to Django syntax. I am still awaiting feedback. Thanks.

**Comment by Nehemiah60** [comment]
Hello, @brylie. Here is my first PR. I will be waiting for your feedback so that I can move to create the factories for the apps with models. Thanks.
https://github.com/CiviWiki/OpenCiviWiki/commit/0f02e6f2af9e5cf55dab27896680acae567b2d5e

**Comment by Nehemiah60** [comment]
Hello @brylie , I made a new PR on this issue [762f44a](https://github.com/CiviWiki/OpenCiviWiki/commit/762f44adff84b1f9d804b1ef1a9227e32857ae23). I also added a factory file for the categories app. If you get time can also go through the threads and accounts app. However, there are issues with duplicated blocks of codes, which I had to comment on both in the threads and accounts app, factory.py files. I would appreciate your help. Thanks.
